packages/api-openfin/notification.html

```diff
@@ -0,0 +1,11 @@
+<h5 id="message">Message</h5>
```

Comment:
![image](https://cloud.githubusercontent.com/assets/9324259/23799648/30acbdbc-05a1-11e7-8769-51a9b32c57a0.png)

This is the output from the notification box, with extra tags for html, body and head that were not included in the template (This is will the changes you suggestd, bar the doctype).